bedrock/settings/static_media.py

```diff
@@ -375,7 +376,6 @@
     },
     'firefox_onboarding_user_actions': {
         'source_filenames': (
-            'css/base/simple_footer-resp.less',
```

Comment:
Note: this less file doesn't seem to be used in either on-boarding bundles, so I'm removing it. Could do with some double-checking to make sure I'm not missing something obvious.